### DISCUSSION TITLE
The Edit Document cursor is always at the end of the story.

### DISCUSSION THREAD

**Discussion by noa7777** [gsantner/markor#2423] [Ideas]
Personally, I prefer that every time I edit a story, the cursor is at the beginning of the story.  There is an option in the settings, and when you load it, the cursor will appear at the end of the article.  But when I cancel this option, the cursor still doesn't appear at the beginning of the article.  It is recommended to add the option that the cursor is always at the beginning of the article.
 Or suggest so.  You can find the position of the first line of text in your line of sight in preview mode, and when you enter edit mode, the cursor is there.

**Comment by gsantner** [comment]
On QuickNote and ToDo (tabs) it is expected behavioir of Markor - so you can **Quick** enter your notes.

For everything else, the cursor position is saved/restored or bottom position for first time opened when you have the option checked.

**Reply by yanayz** [reply]
This "expected" behavior doesn't necessarily make sense to everyone. Letting a user decide would be great (my preference too would be to add at the top of the file for quick note). It's also how Google Keep works when you pin a new note to keep it top of mind.

Would be great if you reconsider this 🙂

**Reply by Silverhawk85** [reply]
I also asked myself why the option in the settings of the app didn't change anything in the todo list. I thought it was simply broken until I read this thread.

Yes, it's a valid argument that you can add notes faster if the cursor always appears at the end but you can also argument, that you always have to scroll all the way up your todo list, if you only want to check todays tasks. So the the question is: What's more worse?

**Reply by mikesautkulis** [reply]
I came here after looking through the settings for a "place cursor at top" option since there's a "place cursor at bottom" option. I'm not exacyin a position to make the change myself but I would love to see it happen. 

Example: I use the "daily-notes" extension in VSCode on desktop which places new dated entries at the top of the document. I use the "QuickNote" button to access this file on mobile, and it would be great to have this option to streamline that flow.

**Comment by noa7777** [comment]
Simply put, there is a very long diary, and I need the first line to write the most recent content, so I need this function
